inst/maintenance/unit/test-audit.R

```diff
@@ -20,14 +20,22 @@ test_that("test audit ode block dadt", {
   expect_silent(mcode("audit2b", code2b, compile = FALSE))
   
   code3 <- "$PLUGIN nm-vars \n $CMT A1 B C  \n $ODE @audit \n DADT(1)  = 0;"
-  expect_error(
+  expect_warning(
```

Comment:
Hmm, so there are some cases in user code where what would have been an error will now just be a warning after this change?

Reply:
Yes. I must have been on edge when writing the NM stuff.

What about reversing this and making it an error all the time now?  I was wondering about this ... it probably should be an error and users can opt out from this audit in fringe cases where each compartment isn't explicitly written into the `$ODE` block (like writing in a loop). 

What do you think?

Reply:
Thanks for confirming.  I don't have a strong leaning there, and don't have any objections to going with the warning if you prefer to go with the less aggressive behavior when consolidating the nm-vars and non-nmvars checks.